internal/schemautil/helpers.go

```diff
@@ -41,15 +41,18 @@ func GetAPIServiceIntegrations(d ResourceStateOrResourceDiff) []aiven.NewService
 	return apiServiceIntegrations
 }
 
-func GetProjectVPCIdPointer(d ResourceStateOrResourceDiff) *string {
+func GetProjectVPCIdPointer(d ResourceStateOrResourceDiff) (*string, error) {
 	vpcID := d.Get("project_vpc_id").(string)
 	var vpcIDPointer *string
-	if len(vpcID) > 0 {
-		parts := strings.SplitN(vpcID, "/", 2)
-		vpcID := parts[1]
-		vpcIDPointer = &vpcID
+
+	parts := strings.SplitN(vpcID, "/", 2)
```

Comment:
can we use `schemautil.SplitResourceID2` instead? it returns `error` already, too

Reply:
With `schemautil.SplitResourceID2` we will get a `invalid resource id: %s` error which might be confusing for a user, that is why I would like to be more specific with the error message